### ISSUE TITLE
How can i read tensorboard by python ?

### ISSUE THREAD

**Issue by Mr-bor** [lanpa/tensorboardX#681]
hi,
i find event_accumulator.EventAccumulator() can help me read a tensorboard, and i know how to read scalar, but how can i read Tensors/text ? i use writer.add_text('A',B) in my pytorch code, and i want to read A and B only by python, because  tensoard is very slow to load my item. And i find A/B are in Tensors.
i think it is a easy thing, but i do not know how to do it.
Thanks!

**Comment by lanpa** [comment]
yes, that's easy. There's example to read the images: https://github.com/lanpa/tensorboard-dumper/blob/master/dump.py

**Comment by Mr-bor** [comment]
oh,i get it, and can you tell me how to get string_val and translate it? 
Thanks!

[TensorEvent(wall_time=1673285639.0386734, step=0, tensor_proto=dtype: DT_STRING
tensor_shape {
  dim {
    size: 1
  }
}
string_val: "\345\212\240\345\205\245zhedang5\357\274\214\345\260\206flag\346\224\271\344\270\2720~2"
)]
